include/TMS/dev/TMP117I2CDevice.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef TMS_TMP117I2CDEVICE_HPP
+#define TMS_TMP117I2CDEVICE_HPP
+
+#include <EVT/io/I2C.hpp>
+#include <TMS/dev/I2CDevice.h>
+#include <TMS/dev/TMP117.hpp>
+
+namespace TMS {
+
+class TMP117I2CDevice : I2CDevice {
+
+public:
+    /**
+         * default constructor for instantiation of uninitialized instances
+         */
+    TMP117I2CDevice();
+
+    /**
+         * TMP117 constructor
+         *
+         * @param[in] tmp117 pointer to tmp object
+         * @param[in] tempValue where the temp value is stored
```

Comment:
This is kind of weird, but I feel like this should be marked as an output parameter because it's being passed in, so data can be pushed out